packages/dnb-eufemia/src/extensions/forms/Field/BankAccountNumber/BankAccountNumber.tsx

```diff
@@ -108,6 +108,9 @@ function BankAccountNumber(props: Props) {
 }
 
 function isValidAccountNumber(digits: string) {
+  if (digits === '00000000000') {
```

Comment:
```suggestion
  if (parseFloat(digits) === 0) {
```

Would be better in terms of supporting other countries.